src/Networks/Rutanio/Rutanio/Networks/Rules/RutanioBugFixPosFutureDriftRule.cs

```diff
@@ -0,0 +1,45 @@
+using Blockcore.Features.Consensus.Rules.CommonRules;
+
+
+namespace Rutanio.Networks.Rules
+{
+    /// <summary>
+    /// A rule that will verify the block time drift is according to the PoS consensus rules for the <see cref="RutanioMain"/> network (and its test networks).
+    /// New networks must use the <see cref="PosFutureDriftRule"/>.
+    /// </summary>
+    public class RutanioBugFixPosFutureDriftRule : PosFutureDriftRule
```

Comment:
you dont need this rule its a stratis only patch just use `PosFutureDriftRule`
Rutanio genesis is far beyond the `DriftingBugFixTimestamp`